app/test/e2e/v2/get-area.spec.js

```diff
@@ -103,9 +103,17 @@ describe('Get areas - V2', () => {
             [{ userId: USERS.USER.id }, { userId: USERS.MANAGER.id }, { userId: USERS.ADMIN.id }]
         );
 
-        const response = await requester.get(`/api/v2/area?loggedUser=${JSON.stringify(USERS.ADMIN)}&all=true`);
+        // First sync areas
+        const response = await requester.post(`/api/v2/area/sync`).send({ loggedUser: USERS.ADMIN });
         response.status.should.equal(200);
-        response.body.should.have.property('data').and.be.an('array').and.have.length(5);
+        response.body.should.have.property('data').and.be.an('object');
+        response.body.data.should.have.property('syncedAreas').and.equal(1);
+        response.body.data.should.have.property('createdAreas').and.equal(2);
+
+        // Get all areas
+        const getResponse = await requester.get(`/api/v2/area?loggedUser=${JSON.stringify(USERS.ADMIN)}&all=true`);
+        getResponse.status.should.equal(200);
+        getResponse.body.should.have.property('data').and.be.an('array').and.have.length(5);
```

Comment:
this needs additional validations

Reply:
Added assertions for the ids returned here!